### PR TITLE
TDP-4111 StartGather/StopGather

### PR DESCRIPTION
**Pull request by muesono-daitan**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

File: voice/about.md [hunk 1]
```diff
@@ -54,12 +56,13 @@
...
+| [DTMF](bxml/callbacks/dtmf.md)                                      | Bandwidth API sends this for every digit detected after a `<StartGather>` is executed. |
+| [Recording Available](bxml/callbacks/recordingAvailable.md)         | Bandwidth API sends this to the application when a recording started by either a `<StartRecording>` or a `<Record>` verb is available for download. |
 | [Transfer Disconnect](bxml/callbacks/transferDisconnect.md)         | Bandwidth API sends this when any leg of a `<Transfer>` ends.                                                                    |
 | [Transcription Available](bxml/callbacks/transcriptionAvailable.md) | Bandwidth API sends this when the transcription of a recording is available for download.                                        |
-| [Conference Completed](bxml/callbacks/conferenceCompleted.md)       | Bandwidth API sends this after the last member leaves the conference.                                                            |
```
Comment (muesono-daitan) [inline voice/about.md]: Changed the order of entries to match the order in other pages.

File: voice/bxml/about.md [hunk 2]
```diff
@@ -53,15 +55,15 @@ BXML callbacks are HTTP messages that are sent to your application server to not
...
+| [Record Complete](callbacks/recordComplete.md)              | Bandwidth API sends this to the application when a `<Record>` is completed. The recording is not available to download until the [Recording Available](callbacks/recordingAvailable.md) event is received.    |
+| [Redirect](callbacks/redirect.md)                           | Bandwidth API sends this to the application when a `<Redirect>` is requested or when a POST request is made to the [/calls/{callId}](../methods/calls/postCallsCallId.md) endpoint requesting a redirect.     |
+| [Transfer Answer](callbacks/transferAnswer.md)              | Bandwidth API sends this to the application when a `<Transfer>` is answered.                                                                                                                                  |
+| [Transfer Complete](callbacks/transferComplete.md)          | Bandwidth API sends this to the application when a `<Transfer>` is complete and the original call needs to continue.                                                                                        |
```
Comment (muesono-daitan) [inline voice/bxml/about.md]: Synced the text in the about pages.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/about.md (hunk 2 above)
Comment (n-miles) [reply voice/bxml/about.md]: I prefer the version with less text but :shrug:

File: voice/bxml/callbacks/dtmf.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+| direction        | The direction of the call. Either `inbound` or `outbound`. The direction of a call never changes. |
+| callId           | The call id associated with the event. |
+| callUrl          | The URL of the call associated with the event. |
+| startTime        | Time the call was started, in ISO 8601 format. |
+| answerTime       | Time the call was answered, in ISO 8601 format. |
+| parentCallId     | (optional) If the event is related to the B leg of a `<Transfer>`, the call id of the original call leg that executed the `<Transfer>`. Otherwise, null. |
+| transferCallerId | (optional) If the event is related to the B leg of a `<Transfer>`, the phone number used as the `from` field of the B-leg call, in E.164 format (e.g. +15555555555). Otherwise, null. |
+| transferTo       | (optional) If the event is related to the B leg of a `<Transfer>`, the phone number used as the `to` field of the B-leg call in E.164 format (e.g. +15555555555). Otherwise, null. |
+| tag              | (optional) The `tag` specified earlier in the call. If no `tag` was specified or it was previously cleared, null. |
+| digit            | The digit collected in the call. |
```
Comment (n-miles) [inline voice/bxml/callbacks/dtmf.md]: Move nearer to the top since it's the most important part?

File: voice/bxml/verbs/startGather.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+{% method %}
+## XML: `<StartGather>`
+The StartGather verb is used to get asynchronous notifications of DTMF digits collected in the call while other verbs are executed.
+
+It can, for instance, listen for DTMF digits while a call is in a [`<Conference>`](conference.md),
+in a [`<Transfer>`](transfer.md), in a [`<Bridge>`](bridge.md), or while executing other verbs.
+
+It cannot be used with the [`<Forward>`](forward.md) verb, and it is paused during the execution of a [`<Gather>`](gather.md) verb.
```
Comment (n-miles) [inline voice/bxml/verbs/startGather.md]:
```suggestion
It cannot be used with the [`<Forward>`](forward.md) verb, and it is paused during the execution of a [`<Gather>`](gather.md) verb; digits pressed during a [`<Gather>`](gather.md) will not be duplicated as DTMF events.
```

File: voice/bxml/callbacks/dtmf.md (hunk 3 above)
Comment (muesono-daitan) [reply voice/bxml/callbacks/dtmf.md]: Moved right after `callId`.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/about.md (hunk 2 above)
Comment (muesono-daitan) [reply voice/bxml/about.md]: re-synced about files.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/startGather.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+{% method %}
+## XML: `<StartGather>`
+The StartGather verb is used to get asynchronous notifications of DTMF digits collected in the call while other verbs are executed.
+
+It can, for instance, listen for DTMF digits while a call is in a [`<Conference>`](conference.md),
+in a [`<Transfer>`](transfer.md), in a [`<Bridge>`](bridge.md), or while executing other verbs.
+
+It cannot be used with the [`<Forward>`](forward.md) verb,
+and it is paused during the execution of a [`<Gather>`](gather.md) verb; digits pressed during a [`<Gather>`](gather.md) will not be duplicated as DTMF events.
```
Comment (marcelohossomi) [inline voice/bxml/verbs/startGather.md]:
```suggestion
It cannot be used with the [`<Forward>`](forward.md) verb.

It is paused during the execution of a [`<Gather>`](gather.md) verb; digits pressed during a [`<Gather>`](gather.md) will not be duplicated as DTMF events.
```

File: voice/bxml/verbs/startGather.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+The StartGather verb can be cancelled by the [`<StopGather>`](stopGather.md) verb.
+
+### Attributes
+| Attribute           | Description |
+|:--------------------|:------------|
+| dtmfUrl             | URL to send the [dtmf](../callbacks/dtmf.md) event to. May be a relative URL. |
```
Comment (marcelohossomi) [inline voice/bxml/verbs/startGather.md]: Looks like we use all-caps for DTMF in other places
```suggestion
| dtmfUrl             | URL to send the [DTMF](../callbacks/dtmf.md) event to. May be a relative URL. |
```

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Review by marcelohossomi — APPROVED** [review]

File: SUMMARY.md [hunk 7]
```diff
@@ -180,6 +182,7 @@
 	 * [Conference Redirect](voice/bxml/callbacks/conferenceRedirect.md)
 	 * [Conference Recording Available](voice/bxml/callbacks/conferenceRecordingAvailable.md)
 	 * [Disconnect](voice/bxml/callbacks/disconnect.md)
+	 * [Dtmf](voice/bxml/callbacks/dtmf.md)
```
Comment (n-miles) [inline SUMMARY.md]: Should this be all caps?
```suggestion
	 * [DTMF](voice/bxml/callbacks/dtmf.md)
```

File: voice/bxml/callbacks/dtmf.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
+{% method %}
+## DTMF Event
+
+The dtmf event is sent for every digit detected after a [`<StartGather>`](../verbs/startGather.md) verb is executed.
```
Comment (n-miles) [inline voice/bxml/callbacks/dtmf.md]:
```suggestion
The DTMF event is sent for every digit detected after a [`<StartGather>`](../verbs/startGather.md) verb is executed.

You may not respond to this event with BXML.
```

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4111.s3-website-us-east-1.amazonaws.com/

**Review by n-miles — APPROVED** [review]

**Pull request merged**